go.mod

```diff
@@ -1,40 +1,14 @@
-module github.com/go-swagno/swagno
+module github.com/domhoward14/swagno
```

Comment:
you should change it back 😄  This change also effect all of your imports in all files. You should also replace them as `github.com/go-swagno/swagno`  😅